eng/pipelines/runtime.yml

```diff
@@ -1208,6 +1243,13 @@ extends:
             extraStepsParameters:
               creator: dotnet-bot
             testRunNamePrefixSuffix: Mono_Release
+            extraVariablesTemplates:
+              - template: /eng/pipelines/common/templates/runtimes/test-variables.yml
+                forwardedParameters:
+                  - osGroup
+                  - osSubgroup
+                  - runtimeFlavor
+                  - runtimeVariant
```

Comment:
Is there a way to avoid this added noise? I see that this is now added to every runtime tests build leg. Can't the build-runtime-tests-and-send-to-helix.yml have this attached?

Reply:
I'll try to figure out a way to make this less noisy.

Reply:
I tried to attach it to build-runtime-tests-and-send-to-helix.yml, but I couldn't get it to work.

The problem is that variables in YAML are a job level concept, but we're trying to add extra variables for a specific step. We can make the extra variables templates have the same parameters passed as the extra steps templates, but it doesn't look like we can attach variables to a step template unless we make our "extra steps" templates into job templates that wrap the global-build-job template, which I don't like.

Reply:
I've done the simplification I can (inline the most common "forwarded parameters" so we don't need to manually forward the common ones), but I can't do any more as you can't templatize what type of template a file is.